getting-started-with-directus-and-flask/index.md

```diff
@@ -0,0 +1,396 @@
+
+Create two text input fields - one with the key of `title` and one `description`.
+
+Navigate to the content module and enter the `global` collection. Collections will generally display a list of items, but as a singleton, it will launch directly into the one-item form. Enter information in the title and description field and hit save.
+
+![Global metadata fields filled with custom text](global-collection-content.png)
+
+By default, new collections are not accessible to the public. Navigate to **Settings** -> **Access Control** -> **Public** and give Read access to the Global collection.
+
+## Creating a Directus Module
+
+Since your data will be fetched via the Directus REST API, you will need to create a module that encapsulates all that logic and exposes a simple interface to the outside world.
+
+To accomplish this, create a *directus.py* file and add the following content to it:
+
+```py
+import requests
+import os
+
+DIRECTUS_BASE_URL = os.environ.get("DIRECTUS_BASE_URL")
+
+
+def get_global_data():
+    response = requests.get(f"{DIRECTUS_BASE_URL}/items/global")
+    return response.json().get("data")
+```
+
+By now this is all you need, but in the following sections, you will also create a new function to fetch data from other collections.
+
+## Rendering The Home Page
+
+To render the site home page, create a new route that uses the directus module to get the global data and use it on a page template.
+
+### Creating Page Templates
+
+First of all, you have to create a base template to be used by all your pages. Create a *templates* directory and a file called *base.html* in it. Now place the following content on this file:
+
+```jinja
+<!DOCTYPE html>
+<html lang="en" data-theme="dark">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link
+			rel="stylesheet"
+			href="https://cdn.jsdelivr.net/npm/@picocss/pico@2/css/pico.min.css"
+		/>
+		{% block head %}{% endblock %}
+
+		<title>{% block title %}Directus blog post with Flask{% endblock %}</title>
+	</head>
+	<body>
+		<header class="container">
+			<nav>
+				<ul>
+					<li>
+						<a href="/"><strong>Home</strong></a>
+					</li>
+				</ul>
+				<ul>
+					<li><a href="/blog">Blog</a></li>
+					<li><a href="/about">About</a></li>
+					<li><a href="/conduct">Code of Conduct</a></li>
+					<li><a href="/privacy">Privacy Policy</a></li>
+				</ul>
+			</nav>
+		</header>
+
+		<main class="container">{% block content %}{% endblock %}</main>
+	</body>
+</html>
+```
+
+:::info
+
+Note there is a stylesheet in this code that imports [Pico](https://picocss.com/), a minimal class-less CSS library used to add some basic styles on the website. It was only used on this article to make things look better, but you can feel free to use whichever library you prefer and even use the default browser styles.
+
+:::
```

Comment:
Put this in the Before You Start section, removing the stylesheet and the nav - which gets added at the end. Can you also update the title to `Directus x Flask`, and remove the `data-theme` from the `html` tag